freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_healthagent_setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+#
+# Add anonymous access only for replication agreements
+#
+export HOSTCN=`hostname -f| sed -E 's/^|\./,dc=/g' | sed 's/^,[^,]*,//g'`
```

Comment:
using `hostname -d` would simplify the sed a bit I think

Reply:
changed.